extensions/amp-bind/amp-bind.md

```diff
@@ -98,7 +98,7 @@ For AMP components, only select attributes are bindable:
 | Component | Attributes |
 | --- | --- |
 | amp-carousel | slide |
-| amp-img | src, srcset, alt |
+| amp-img | src, srcset, alt, height, width |
```

Comment:
@choumx - Are the [common attributes](https://www.ampproject.org/docs/reference/common_attributes) for AMP components bindable?  If so, then as you say, remove `height` and `width` and replace this sentence: 

"For AMP components, only select attributes are bindable" 

with this:

 "For AMP components, the [common attributes](https://www.ampproject.org/docs/reference/common_attributes) are bindable along with the following specific attributes:"